Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -210,6 +211,16 @@ namespace RedirectOutput
         });
 
         SetRedirection("stderr", g_redirect_stderr_saved, g_redirect_stderr, []([[maybe_unused]] const char* msg) {
+            AZStd::string lastPythonError = msg;
+            constexpr const char* pythonErrorPrefix = "ERROR:root:";
+            constexpr size_t lengthOfErrorPrefix = AZStd::char_traits<char>::length(pythonErrorPrefix);
+            auto errorPrefix = lastPythonError.find(pythonErrorPrefix);
+            if (errorPrefix != AZStd::string::npos)
+            {
+                lastPythonError.erase(errorPrefix, lengthOfErrorPrefix);
+            }
```

Comment:
FYI you can use [AZStd::erase](https://en.cppreference.com/w/cpp/string/basic_string/erase2) for this.

```
AZStd::string lastPythonError = msg;
AZStd::erase(lastPythonError, "ERROR:root");
```